java-manta-client/src/main/java/com/joyent/manta/http/EncryptionHttpHelper.java

```diff
@@ -251,15 +254,21 @@ public MantaObjectInputStream httpRequestAsInputStream(final HttpUriRequest requ
         }
 
         final Long initialSkipBytes;
-        final Long plaintextRangeLength;
+        Long plaintextRangeLength;
+        final Long plaintextStart;
+        final Long plaintextEnd;
 
         if (hasRangeRequest) {
             Long[] rangeProperties = calculateSkipBytesAndPlaintextLength(request, requestHeaders);
```

Comment:
I thought we already had a ticket to clean this up as a class instead of array, but I can't seem to find it.